### PR TITLE
Edted README

### PR DESCRIPTION
**Pull request by RichardLitt**
Fixed badges and https issues, small edits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +145,8 @@ Contributions welcome. Please check out [the issues](https://github.com/multifor
 
 Check out our [contributing document](https://github.com/multiformats/multiformats/blob/master/contributing.md) for more information on how we work, and about contributing in general. Please be aware that all interactions related to multiformats are subject to the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
 
-Small note: If editing the Readme, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
+Small note: If editing the README, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
+
 ## License
 
-[MIT](LICENSE) © David Dias
+[MIT](LICENSE) © 2015 David Dias
```
Comment (diasdavid) [inline README.md]: 2017?

Comment (VictorBjelkholm) [inline README.md]: Should be the year it started being active (the license) rather than the current year, so I guess this is fine.

Comment (diasdavid) [inline README.md]: Understood, thought it was the other way around :)

**Comment by RichardLitt** [comment]
That was in the license file :)

**Pull request merged**